src/plugins/specload/README.md

```diff
@@ -0,0 +1,73 @@
+- infos = Information about the specload plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs = quickdump
+- infos/provides =
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = nodep libc configurable experimental unfinished
+- infos/metadata =
+- infos/description = loads a specification from an external application
+
+## Introduction
+
+In order to optimally use Elektra an application has to provide a specification. This specification has to be mounted in the `spec`
+namespace by the user (or automatically during the install process). However, this specification should in most cases not be modified by
+the user. If it is modified the user has to be sure it is still compatible to the original specification. Otherwise the application might
+crash because it e.g. expected a default value to be provided by the specification that the user changed or removed.
+
+This is were this plugin comes in. Unlike other storage plugins `specload` doesn't use the file given during mounting for most of the
+configuration. Instead `specload` requests the specification with which an application was compiled from the application itself. On top
+of this base specifications the user can make some modifications, which are the only keys stored in the mounted file. All modifications
+made by the user are verified by the plugin. The user can never modify the specification in a way to would break application compatibility.
+
+NOTE: currently the modifications a user can make are very limited. See [Limitations](#limitations) below.
+
+## Usage
+
+To use `specload` first make sure Elektra was compiled with `BUILD_SHARED` enabled, and the `quickdump` plugin is available as well:
```

Comment:
This is dependency, not usage. Maybe we should swap Dependency and Usage sections?